spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -72,4 +72,199 @@
       expect(page).to have_current_path(idv_hybrid_handoff_url)
     end
   end
+
+  describe 'navigating to How To Verify from Agreement page in 50/50 state' do
+    before do 
+      allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+    end
+
+      context 'opt in false at start but true during navigation' do
+        before do
+          allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+
+          sign_in_and_2fa_user
+          complete_doc_auth_steps_before_agreement_step
+          complete_agreement_step
```

Comment:
Your changes look good! Another refactoring suggestion: You could pull all these before blocks out to join the one on line 77, and replace it with a let variable for the opt_in_enabled value (and `complete_how_to_verify_step` where needed)
```suggestion
(in the block above)
          allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { opt_in_enabled }

          sign_in_and_2fa_user
          complete_doc_auth_steps_before_agreement_step
          complete_agreement_step
          
          (in this block)
          let(:opt_in_enabled) { false }
```